web/src/message_list_view.js

```diff
@@ -127,7 +127,7 @@ function render_group_display_date(group, message_container) {
 function update_group_date(group, message_container, prev) {
     // Mark whether we should display a date marker because this
     // message has a different date than the previous one.
-    group.date_unchanged = same_day(message_container, prev);
+    group.date_unchanged = same_day(message_container?.msg, prev?.msg);
```

Comment:
Can `message_container` her be undefined? I would have thought only `prev` could be.

Reply:
I don't remember why I did this (now that it's been a little while since I put up this PR) but I'll make sure to clean this up if needed as I convert it to typescript, so I'm not too worried about figuring out the answer right now.